### PR TITLE
* #173: Change GitHub uploading assets to an optional step.

### LINKED ISSUE #173: Add option to release on the GitHub without assets
## Problem

Some releases don't need to have assets attached. We want to skip this step sometimes.

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #173

### CODE REVIEW

File: doc/changes/changes_0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Exasol Release Droid 0.7.0, released 2021-??-??
+
+Code name: 
+
+## Summary
+
+Uploading assets for a GitHub release now an optional step an only work if your repository has a workflow file `release_droid_upload_github_release_assets.yml`.
```
Comment (morazow) [inline doc/changes/changes_0.7.0.md:7]:
```suggestion
Uploading assets for a GitHub release now an optional step and only works if your repository has a workflow file `release_droid_upload_github_release_assets.yml`.
```

File: src/main/java/com/exasol/releasedroid/adapter/github/GitHubRelease.java [hunk 2]
```diff
@@ -63,6 +65,15 @@ public String getRepositoryName() {
         return this.repositoryName;
     }
 
+    /**
+     * Check if upload assets required.
+     *
+     * @return true if upload assets required
+     */
+    public boolean uploadAssets() {
```
Comment (morazow) [inline src/main/java/com/exasol/releasedroid/adapter/github/GitHubRelease.java:73]: Maybe it is good idea to prefix this with `is` or `has`, it is clearer for boolean checks.

**Review by morazow — DISMISSED** [review]
Approved with minor edits and  a suggestion

File: src/main/java/com/exasol/releasedroid/adapter/github/GitHubRelease.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/releasedroid/adapter/github/GitHubRelease.java:73]: Good idea, done!

**Pull request merged**